lambdas/tests/unit/services/session-request-validator.test.ts

```diff
@@ -182,11 +183,48 @@ describe("session-request-validator.ts", () => {
             ).rejects.toThrow(
                 expect.objectContaining({
                     message: "Session Validation Exception",
-                    details: "Invalid scope parameter",
+                    details: "Invalid request: scoringPolicy in evidence_requested does not equal gpg45",
                 }),
             );
         });
 
+        it("should pass when the evidence_requested is included and scoringPolicy is gpg45", async () => {
+            const client_id = "request-client-id";
+            const redirect_uri = "redirect-uri";
+            const state = "state";
+
+            jest.spyOn(jwtVerifier.prototype, "verify").mockReturnValue(
+                await Promise.resolve({
+                    client_id: client_id,
+                    redirect_uri: redirect_uri,
+                    state: state,
+                    evidence_requested: {
+                        scoringPolicy: "gpg45",
+                    },
+                    shared_claims: personIdentity,
+                } as JWTPayload),
+            );
+
+            await expect(async () => sessionRequestValidator.validateJwt(Buffer.from("test-jwt"), client_id)).resolves;
+        });
+
+        it("should pass when the there is no evidence_requested", async () => {
+            const client_id = "request-client-id";
+            const redirect_uri = "redirect-uri";
+            const state = "state";
+
+            jest.spyOn(jwtVerifier.prototype, "verify").mockReturnValue(
+                await Promise.resolve({
+                    client_id: client_id,
+                    redirect_uri: redirect_uri,
+                    state: state,
+                    shared_claims: personIdentity,
+                } as JWTPayload),
+            );
+
+            await expect(async () => sessionRequestValidator.validateJwt(Buffer.from("test-jwt"), client_id)).resolves;
+        });
+
         xit("should fail to validate the jwt if scope is missing", async () => {
```

Comment:
We should remove the commented test if it is about scope

Reply:
I have removed all references of scope 👍